index.yaml

```diff
@@ -73,6 +73,17 @@ distributions:
       ubv8: foxy/release-focal-arm64-build.yaml
     source_builds:
       default: foxy/source-build.yaml
+  foxy:
+    ci_builds: {}
```

Comment:
I assume you plan to add them after the first round of Debian packages has been built successfully>

Reply:
Your assumption is correct. The pull request description includes "I'll port CI job configurations from Foxy as a follow-up next week." as an indication toward this.